src/operator/tensor/matrix_op.cc

```diff
@@ -1038,6 +1038,7 @@ Example::
 
 NNVM_REGISTER_OP(_split_v2)
 .add_alias("_npi_split")
+.add_alias("_npi_array_split")
```

Comment:
you do not need to re-alias for this case, simply call `_npi.split` directly in the Python wrappers.